data_types/softlayer_item_price.go

```diff
@@ -10,3 +11,7 @@ type Item struct {
 	Description string `json:"description"`
 	Capacity    string `json:"capacity"`
 }
+
+type Category struct {
+	CategoryCode string `json:"categoryCode"`
```

Comment:
Categories field in Softlayer_Item_Price can be empty, but if we have one Category listed, the categorycode must be provided. it's a mandatory field at present.